__tests__/src/components/BackgroundPluginArea.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { BackgroundPluginArea } from '../../../src/components/BackgroundPluginArea';
+
+it('should render outer element correctly', () => {
+  const wrapper = shallow(<BackgroundPluginArea />);
+  expect(wrapper.find('.mirador-background-plugin-area').length).toBe(1);
+});
+
+it('should render plugin components', () => {
+  /** */
+  const PluginComponentA = props => <div>A</div>;
+  /** */
+  const PluginComponentB = props => <div>B</div>;
+
+  const wrapper = shallow(
+    <BackgroundPluginArea
+      PluginComponents={[PluginComponentA, PluginComponentB]}
+    />,
+  );
+
```

Comment:
May be there could be a check again, if the BackgroundPluginArea is rendered as well, when passing PluginComponents, like in the first test.